utils/resource/usage.go

```diff
@@ -230,14 +280,17 @@ func (p *proc) getActiveUsage(secondsSinceLastUpdate float64) (float64, float64,
 		if totalCPU > p.lastTotalCPU {
 			newCPU := totalCPU - p.lastTotalCPU
 			cpu = newCPU / secondsSinceLastUpdate
+			p.cpuCycles = newCPU
 		}
 		if io.ReadBytes > p.lastReadBytes {
 			newRead := io.ReadBytes - p.lastReadBytes
 			read = float64(newRead) / secondsSinceLastUpdate
+			p.read = newRead
```

Comment:
This doesn't return the number of reads. This is the amount of bytes read since the last update.

Reply:
updated.